From 8ec72f5f823edaea9e564207b9999d75a8de80bd
ID: Phong
Reflects refactoring of JNI helper code that depends on private
methods within libnativehelper.
Sunday 28 June 2020
Bug : 90 95
Monday 29 June 2020
Bug : 
Tueday 30 June 2020
Bug : 32 61
Change-Id: I2273fc26bb91886f3a94105cc3c12824fd0ded8e
Merged-In: I2273fc26bb91886f3a94105cc3c12824fd0ded8e
Exempt-From-Owner-Approval: cherry pick
(cherry picked from commit 04f051c926ee5b3a475243f1b162de4fe687ee37)
diff --git a/libdataloader/DataLoaderConnector.cpp b/libdataloader/DataLoaderConnector.cpp
index 6c4a693..c0ade23 100644
--- a/libdataloader/DataLoaderConnector.cpp
+++ b/libdataloader/DataLoaderConnector.cpp
@@ -16,7 +16,7 @@
 #define LOG_TAG "incfs-dataloaderconnector"
 
 #include <android-base/logging.h>
-#include <nativehelper/JNIHelp.h>
+#include <nativehelper/JNIPlatformHelp.h>
 #include <nativehelper/scoped_local_ref.h>
 #include <nativehelper/scoped_utf_chars.h>
 #include <sys/stat.h>
